src/resolve/sourceComponent.ts

```diff
@@ -259,10 +259,8 @@ export class SourceComponent implements MetadataComponent {
       delete this.destructiveChangesType;
     } else {
       this.markedForDelete = true;
-      // eslint-disable-next-line no-unused-expressions
-      destructiveChangeType === DestructiveChangesType.PRE
-        ? (this.destructiveChangesType = DestructiveChangesType.PRE)
-        : (this.destructiveChangesType = DestructiveChangesType.POST);
+      this.destructiveChangesType =
```

Comment:
I had to stare at this one for a second.  comment could be
// [POST, true, undefined ] => POST.  Only PRE means PRE